### PR TITLE
Improve handling of AWS overview file

### PR DESCRIPTION
**Pull request by lacunoc**
* Make AWS instances overview editable for all users
* Open AWS instances overview file for read and write

### CODE REVIEW

**Comment by lacunoc** [comment]
retest this please

File: igvm/vm.py [hunk 1]
```diff
@@ -1005,19 +1006,33 @@ def aws_get_instances_overview(
                 etag = None
 
             if file.exists() and etag_file.exists() and etag:
-                with open(AWS_INSTANCES_OVERVIEW_FILE_etag) as f:
+                with open(AWS_INSTANCES_OVERVIEW_FILE_etag, 'r+') as f:
```
Comment (kofrezo) [inline igvm/vm.py:1009]: `AWS_INSTANCES_OVERVIEW_FILE_etag` should be `AWS_INSTANCES_OVERVIEW_FILE_ETAG`

File: igvm/vm.py [hunk 2]
```diff
@@ -1005,19 +1006,33 @@ def aws_get_instances_overview(
...
             if etag:
-                with open(AWS_INSTANCES_OVERVIEW_FILE_etag, 'w') as f:
+                with open(AWS_INSTANCES_OVERVIEW_FILE_etag, 'w+') as f:
                     f.write(etag)
-            with open(AWS_INSTANCES_OVERVIEW_FILE, 'w') as f:
+                    os.chmod(AWS_INSTANCES_OVERVIEW_FILE_etag,
+                             stat.S_IREAD |
+                             stat.S_IWRITE |
+                             stat.S_IRGRP |
+                             stat.S_IWGRP |
```
Comment (kofrezo) [inline igvm/vm.py:1023]: Do we really need write permission for the group ?

File: igvm/vm.py [hunk 3]
```diff
@@ -1005,19 +1006,33 @@ def aws_get_instances_overview(
...
+                             stat.S_IROTH |
+                             stat.S_IWOTH)
```
Comment (kofrezo) [inline igvm/vm.py:1025]: Why do we need to make the file world read and also world writable - can't we avoid that ?

**Review by kofrezo — CHANGES_REQUESTED** [review]

File: igvm/vm.py (hunk 1 above)
Comment (lacunoc) [reply igvm/vm.py]: renamed

File: igvm/vm.py (hunk 3 above)
Comment (lacunoc) [reply igvm/vm.py]: We can. I've added the file to `adm` group and removed the world writable flag. Reading does no harm

File: igvm/vm.py (hunk 2 above)
Comment (lacunoc) [reply igvm/vm.py:1025]: We do, since another user needs to be able to edit this file. I've added `adm` group with writable rights.

**Review by kofrezo — APPROVED** [review]

**Comment by lacunoc** [comment]
retest this please

**Pull request merged**